app/controllers/google_sign_in/authorizations_controller.rb

```diff
@@ -1,6 +1,8 @@
 require 'securerandom'
 
 class GoogleSignIn::AuthorizationsController < GoogleSignIn::BaseController
+  skip_forgery_protection only: :create
```

Comment:
I thought about simply removing:
https://github.com/basecamp/google_sign_in/blob/47ec9a7c8a23e907020a95d638da15e4acddea3c/app/controllers/google_sign_in/base_controller.rb#L4

But I thought that in some versions of Rails, this is done by default, and it could also cover other `POST` actions that might be added to this library in the future, and that do need to be protected from forgery.